docs/interacting/bydbctl/bydbctl.md

```diff
@@ -0,0 +1,20 @@
+# bydbctl
+`bydbctl` is the command line tool for interacting with BanyanDB. It is a powerful tool that can be used to create, update, read, and delete schemas. It can also be used to query data stored in streams, measures, and properties.
+
+These are several ways to install:
+
+* Get binaries from [download](https://skywalking.apache.org/downloads/).
```

Comment:
I can't find the ctl download link here.

Do we need a website update?

Reply:
They are in the same tar with the released server binary

<img width="312" alt="image" src="https://github.com/user-attachments/assets/3b037e2a-9b33-4ddb-a802-b587fdc30887">

We should add this detail here.

Reply:
I don't mean svn path. I mean we don't have download link directly from the website.

We should add it.

Reply:
No one but ourselves knows they are released in one place.

@wankai123 Could you add bydctl download project at this page? And target to the correct version and links.

Reply:
https://github.com/apache/skywalking/issues/12510 creates for @hanahmily 

This relies on new release process and scripts.